src/package/index.js

```diff
@@ -53,8 +53,7 @@ class Package {
 
     const exports = {}
     if (!json.exports) {
-      if (!json.main) exports['.'] = { import: this.file(toURL('./index.js')) }
-      exports['.'] = { import: this.file(toURL(json.main)) }
+      exports['.'] = { import: this.file(toURL(json.main || './index.js')) }
```

Comment:
noticed the above logic was a bit broken so I fixed it while in here, the `if` has no effect and the second line would be calling `toURL(undefined)` if there's no `json.main`.